src/components/activity-groups-list/activity-group-item/activity-group-amount-change/activity-group-amount-change.tsx

```diff
@@ -42,7 +42,7 @@ export const ActivityGroupAmountChange: FC<Props> = ({ group }) => {
       let exchangeRate = 0;
 
       if (isString(address)) {
-        exchangeRate = exchangeRates.data[address];
+        exchangeRate = exchangeRates.data[getTokenSlug({ address, id })];
```

Comment:
could you reuse `useTokenExchangeRate` hook here?

Reply:
I'm afraid there is no way to do it because it's a loop inside a hook